src/qt/transactiontablemodel.cpp

```diff
@@ -320,6 +320,9 @@ QString TransactionTableModel::formatTxStatus(const TransactionRecord *wtx) cons
     case TransactionStatus::Abandoned:
         status = tr("Abandoned");
         break;
+    case TransactionStatus::AssumedConfirmed:
+        status = tr("%1 confirmations, pending verification of historical blocks").arg(wtx->status.depth);
```

Comment:
It's not really the historical blocks being verified as much as it is the snapshot...

Maybe just

```suggestion
        status = tr("%1 confirmations, pending verification").arg(wtx->status.depth);
```